Update google spreadsheets docs (#105)
segmentstream-com/segmentstream-com.github.io: _docs/datasources/google sheets.md

  <li><a href="#display-name">Display name</a></li>
  <li><a href="#google-spreadsheet-url">Google Spreadsheet URL</a></li>
  <li><a href="#range">Range</a></li>
  <li><a href="#column-types">Column types</a></li>
  <li><a href="#destination-table-name">Destination table name</a></li>
  <li><a href="#saving-disconnecting-disabling">Saving, Disconnecting, Disabling data source</a></li>
</ul>

### <a name="importing-data"></a>Importing data from Google Sheets
------

After enabling this data source, Google Sheets data will be uploaded to BigQuery once every 24 hours.

### <a name="connecting-and-configuring"></a>Connecting and configuring
------

The process of connecting data sources is described in detail in the [overview](/datasources/index).

After authorization you need to set the data source parameters.

### <a name="options-description"></a>Options description
------
![](/img/google-sheets-1.png)

### <a name="display-name"></a>Display name
The name of the data source. It is displayed in the interface in the list of sources.

### <a name="google-spreadsheet-url"></a>Google Spreadsheet URL
Link to to Google Spreadsheet.

> Just copy-paste it from browser URL of spreadsheet table.

Reviewer: ```suggestion
Just copy-paste it from the browser address bar of the spreadsheet table.
```

If you want load data from specific sheet, select it before copying URL.

> Notice! If range not specified, first row of the sheet will be used for field names.

> Notice! Field names must contain only letters, numbers, and underscores, start with a letter or underscore, and be at most 128 characters long.

### <a name="range"></a>Range

> If you want load data only from specific range, type it here using A1 notation.

Reviewer: ```suggestion
If you want to load data only from a specific range, type it here using A1 notation.
```

> Notice! First row of range will be used for field names.

### <a name="column-types"></a>Column types
Optionally you can specify column types.

> For example see screenshot it show how to import costs data from google spreadsheets.

Reviewer: ```suggestion
The screenshot above displays an example of a Google spreadsheet costs data import setup.
```

> If column types not specified, STRING type will be used as default.

Reviewer: ```suggestion
If column types are not specified, STRING type will be used by default.
```
